.github/ISSUE_TEMPLATE/change-author-list.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Change the list of authors
+about: Add, remove, or reorder the authors on a paper
+title: Change authors on Anthology ID XXX
+labels: correction, author-list
+assignees: xinru1414
```

Comment:
That should be anthology-assist now, correct?

Reply:
Correct